app/views/admin/users/index.html.erb

```diff
@@ -53,69 +53,6 @@
 
 </div>
 
-<% @courses.each { |course| %>
```

Comment:
This block may still be useful if we need to list all users.  It can be a function that is requested on the page.  For now it is fine to delete it.

Reply:
yes, we can revert the commit if needed later.